File: Selenium/Activities/Activity3.java
package Activity;

import io.github.bonigarcia.wdm.WebDriverManager;
import org.openqa.selenium.By;
import org.openqa.selenium.WebDriver;
import org.openqa.selenium.firefox.FirefoxDriver;

public class Activity3 {
    public static void main(String[] args) throws InterruptedException {

        System.setProperty(FirefoxDriver.SystemProperty.BROWSER_LOGFILE, "/dev/null");
        WebDriverManager.firefoxdriver().setup();
        WebDriver driver = new FirefoxDriver();
        driver.get("https://v1.training-support.net/selenium/login-form");
        String element = driver.getTitle();
        System.out.println("Home Page Title is " + element);
        driver.findElement(By.xpath("//input[@id ='username']")).sendKeys("admin");
        driver.findElement(By.xpath("//input[@id ='password']")).sendKeys("password");
        driver.findElement(By.xpath("//button[text()='Log in']")).click();
        String message=driver.findElement(By.xpath("//div[@id='action-confirmation']")).getText();
        System.out.println("The display message is " + message);

        Thread.sleep(2000);
        driver.quit();

    }
}

File: Selenium/Activities/Activity3.py
# Import webdriver from selenium
from selenium import webdriver
from selenium.webdriver.common.by import By
from selenium.webdriver.firefox.service import Service as FirefoxService
from webdriver_manager.firefox import GeckoDriverManager

# Set up the Firefox Driver with WebDriverManger
service = FirefoxService(GeckoDriverManager().install())
# Start the Driver
with webdriver.Firefox(service = service) as driver:
    # Open the browser to the URL
    driver.get("https://v1.training-support.net/selenium/login-form")
    print("Home Page Title is " ,driver.title)
    driver.find_element(By.XPATH,"//input[@id ='username']").send_keys("admin")
    driver.find_element(By.XPATH, "//input[@id ='password']").send_keys("password")
    driver.find_element(By.XPATH, "//button[text()='Log in']").click()

    message = driver.find_element(By.XPATH, "//div[@id='action-confirmation']")
    print("The Display Message is " , message.text)
    driver.close()